amy/dashboard/views.py

```diff
@@ -229,7 +234,33 @@ def training_progress(request):
     progress_welcome = get_passed_or_last_progress(request.user, "Welcome Session")
     progress_demo = get_passed_or_last_progress(request.user, "Demo")
 
-    if request.method == "POST":
+    context = {
+        "title": "Your training progress",
+        "progress_training": progress_training,
+        "progress_get_involved": progress_get_involved,
+        "progress_welcome": progress_welcome,
+        "progress_demo": progress_demo,
+    }
+    return render(request, "dashboard/training_progress.html", context)
+
+
+class GetInvolvedCreateView(LoginRequiredMixin, RedirectSupportMixin, AMYCreateView):
+    # permission_required = TODO
+    model = TrainingProgress
+    form_class = GetInvolvedForm
+    template_name = "get_involved_form.html"
+    success_url = reverse_lazy("training-progress")
+    success_message = (
+        "Thank you. Your Get Involved submission will be reviewed within 7-10 days."
```

Comment:
Almost the same message ("Your submission will be evaluated within 7-10 days."). Optionally you could store the "7-10 days" part somewhere as a configurable.

Reply:
I think the slight extra detail is useful given that this message appears at the top of the page and not in the 'Get Involved' section - though I have changed 'reviewed' to 'evaluated' in one of the other PRs for consistency.

I'll leave the 7-10 days part as it is for now - I don't anticipate us changing this in the near future.